atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/impl/AtlasDbServicesImpl.java

```diff
@@ -56,7 +69,31 @@ public void register(Object resource) {
                     }
                 },
                 true);
-        return new AtlasDbServicesImpl(tm);
+
+        KeyValueService kvs = tm.getKeyValueService();
+        TransactionService transactionService = TransactionServices.createTransactionService(kvs);
+        SweepStrategyManager sweepStrategyManager = SweepStrategyManagers.createDefault(kvs);
+        CleanupFollower follower = CleanupFollower.create(ImmutableSet.<Schema>of());;
+        SweepTaskRunner sweepRunner = new SweepTaskRunnerImpl(
+                tm,
+                kvs,
+                new Supplier<Long>() {
+                    @Override
+                    public Long get() {
+                        return tm.getUnreadableTimestamp();
+                    }
+                },
+                new Supplier<Long>() {
+                    @Override
+                    public Long get() {
+                        return tm.getImmutableTimestamp();
+                    }
+                },
```

Comment:
yeah, lets hold off on switching to Java 8 for a bit as that's a big breaking change. I *think* we should be ok, but not sure of all consumers.